File: MDEV1001-M2023-ICE8/Android/app/src/main/java/com/kanwarabhijitsingh/ice8/Movie.kt
package com.kanwarabhijitsingh.ice8

import com.squareup.moshi.Json

data class Movie(
	@Json(name = "_id") val id: String,
	@Json(name = "movieID") val movieID: String,
	val title: String,
	val studio: String,
	val genres: List<String>,
	val directors: List<String>,
	val writers: List<String>,
	val actors: List<String>,
	val year: Int,
	val length: Int,
	@Json(name = "shortDescription") val description: String?,
	val mpaRating: String,
	val criticsRating: Double,
	var isExpanded: Boolean = false
)

File: MDEV1001-M2023-ICE8/iOS/ICE8/ICE8/Movie.swift
//
//  Movie.swift
//  ICE8
//
//  Created by Abhijit Singh on 2023-07-05.
//

struct Movie: Codable
{
    let _id: String
    let movieID: String?
    let title: String
    let studio: String
    let genres: [String]
    let directors: [String]
    let writers: [String]
    let actors: [String]
    let year: Int
    let length: Int
    let shortDescription: String
    let mpaRating: String
    let criticsRating: Double
}
